content/docs/querying/api.md

```diff
@@ -71,6 +71,7 @@ URL query parameters:
 
 - `query=<string>`: Prometheus expression query string.
 - `time=<rfc3339 | unix_timestamp>`: Evaluation timestamp. Optional.
+- `timeout=<duration>`: Evaluation timeout. Optional. Defaults to value of `-query.timeout` flag.
```

Comment:
@brian-brazil I think I'd want a default of 1 or 2 minutes, then per-query configurable timeouts, but which are still capped at something reasonable. So that'd require two flags (default query limit, absolute max limit).